lib/thunder/pages/thunder_page.dart

```diff
@@ -111,6 +112,8 @@ class _ThunderState extends State<Thunder> {
       BlocProvider.of<DeepLinksCubit>(context).handleInitialURI();
       BlocProvider.of<NotificationsCubit>(context).handleNotifications();
     });
+
+    BackButtonInterceptor.add(_handleBackButtonPress);
```

Comment:
Do you need to clean this up in `dispose`?

Reply:
Done!